README.md

```diff
@@ -1442,6 +1444,14 @@ Note: `NODE_ENV` is properly configured by the `next` subcommands, if absent, to
 
 Note: we recommend putting `.next`, or your [custom dist folder](https://github.com/zeit/next.js#custom-configuration), in `.gitignore` or `.npmignore`. Otherwise, use `files` or `now.files` to opt-into a whitelist of files you want to deploy, excluding `.next` or your custom dist folder.
 
+### Security Suggestions
+
+There are some best practices that Next.js supports that will help secure your site:
+
+#### Content Security Policy
+
+Next.js *mostly* supports [CSP](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP). You can use the following CSP policy to get started: `default-src 'self'; style-src 'self' 'unsafe-inline';`
```

Comment:
I think `unsafe-inline` is a bad suggestion for CSP and not needed, since you could define a hash for the inline script (`NextScript`).

Reply:
This is inline **style**. Completely different issue. Trust me. <style> blocks are just randomly added and removed from the DOM. It takes some special logic to get that working.

Reply:
I’m going to see if I can get automatic CSP support in Nexjs including nonces for generates styles.